doc-content/enterprise-only/fuse/installing-on-fuse-eap-proc.adoc

```diff
@@ -1,20 +1,13 @@
 [id='installing-on-fuse-eap-proc']
 = Installing {FUSE} on {EAP_LONG}
-Install {FUSE_LONG} {FUSE_VERSION} on {EAP} 7.2.7 to use with {PRODUCT} to deploy integrated services where required.
-
-[NOTE]
-====
-{FUSE_LONG} 7.7 is supported on {EAP} 7.2, whereas {PRODUCT} 7.8.x and newer is supported on Red Hat {EAP} 7.3. To ensure a stable deployment environment, install {FUSE} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
-
-{PRODUCT} 7.8.x and newer components on {EAP} 7.2 are not officially supported. However, Red Hat provides guidance on a commercially reasonable basis until {FUSE} is available on {EAP} 7.3. `MarshallingFormat.JSON` is not supported on {FUSE} integration with {EAP} 7.2 using `kie-camel`. The default is JAXB.
-====
+Install {FUSE_LONG} {FUSE_VERSION} on {EAP} {FUSE_EAP_VERSION} to use with {PRODUCT} to deploy integrated services where required.
 
 .Prerequisites
 * A {PRODUCT} installation on {EAP_LONG} {EAP_VERSION} is available. For installation instructions, see {URL_INSTALLING_AND_CONFIGURING}#assembly-install-on-eap[_{INSTALLING_ON_EAP}_].
-* A separate instance of {EAP_LONG} 7.2.7 is available.
+* A separate instance of {EAP_LONG} {FUSE_EAP_VERSION} is available.
 
 .Procedure
-. Install {FUSE_LONG} {FUSE_VERSION} on {EAP_LONG} 7.2.7. For installation instructions, see the https://access.redhat.com/documentation/en-us/red_hat_fuse/{FUSE_VERSION}/html-single/installing_on_jboss_eap/index[_Installing on JBoss EAP_].
+. Install {FUSE_LONG} {FUSE_VERSION} on {EAP_LONG} {FUSE_EAP_VERSION}. For installation instructions, see the https://access.redhat.com/documentation/en-us/red_hat_fuse/{FUSE_VERSION}/html-single/installing_on_jboss_eap/index[_Installing on JBoss EAP_] section in {FUSE_LONG} documentation.
 
 . Open the `pom.xml` file in the {FUSE} home directory in a text editor.
 . Create the integration project with a dependency on the `kie-camel` component by editing the `pom.xml` file as shown in the following example:
```

Comment:
I know it's not part of your effort on this task, but I notice that most of the snippets throughout this docs aren't labeled and again would greatly benefit from them. Not your job here, but in case any assemblies in this title end up in your stewardship, then I just suggest that again as a general feedback. Down the road.

Reply:
Thanks, I have put this into https://issues.redhat.com/browse/BXMSDOC-7163 . I have some question as to whether this might break up the procedure visually, but let us cross this bridge when we reach it.